test/cuda_sims.jl

```diff
@@ -189,7 +189,7 @@ end
          a = 0.94,
          ν = 182.5)
 
-  tₑ = 10.0
+  tₑ = 1.0
```

Comment:
Is this enough time to see the formation and stabilization of at least one ridge?

Reply:
A bump forms at 2.5 so I'll crank it back up.